openlibrary/plugins/openlibrary/code.py

```diff
@@ -1241,7 +1245,7 @@ def GET(self):
             query = i.get('data', '')
             data = fulltext_search(query)
             hits = data.get('hits', [])
-            if not hits['hits']:
+            if not hits:
```

Comment:
This change seems to address an untracked bug, and can possibly be made on `master` as well.